python/test/t_OrthogonalBasis_std.expout

```diff
@@ -1,4 +1,14 @@
-basis= class=OrthogonalProductPolynomialFactory univariate polynomial collection=[class=OrthogonalUniVariatePolynomialFamily implementation=class=LaguerreFactory k=2.5 measure=class=Gamma name=Gamma dimension=1 k=3.5 lambda=1 gamma=0,class=OrthogonalUniVariatePolynomialFamily implementation=class=LegendreFactory measure=class=Uniform name=Uniform dimension=1 a=-1 b=1,class=OrthogonalUniVariatePolynomialFamily implementation=class=HermiteFactory measure=class=Normal name=Normal dimension=1 mean=class=Point name=Unnamed dimension=1 values=[0] sigma=class=Point name=Unnamed dimension=1 values=[1] correlationMatrix=class=CorrelationMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[1]] measure=class=ComposedDistribution name=ComposedDistribution dimension=3 copula=class=IndependentCopula name=IndependentCopula dimension=3 marginal[0]=class=Gamma name=Gamma dimension=1 k=3.5 lambda=1 gamma=0 marginal[1]=class=Uniform name=Uniform dimension=1 a=-1 b=1 marginal[2]=class=Normal name=Normal dimension=1 mean=class=Point name=Unnamed dimension=1 values=[0] sigma=class=Point name=Unnamed dimension=1 values=[1] correlationMatrix=class=CorrelationMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[1]
+basis= OrthogonalProductPolynomialFactory
+- measure=Distribution
+- isOrthogonal=true
+- enumerateFunction=class=LinearEnumerateFunction dimension=3
+
+| Index | Type |
+|-------|------|
+|     0 | class=LaguerreFactory k=2.5 measure=class=Gamma name=Gamma dimension=1 k=3.5 lambda=1 gamma=0 |
```

Comment:
Compute the column width so that all columns are equal. Keep only the base polynomial type, removing the details of the parameters. Create a new column with the measure.

Reply:
Fixed at 34040e6c267663e37eb5e6a960b39fc3fb0cdd5a